### PR TITLE
Fix HostMetricsResponse unit test

### PR DESCRIPTION
**Pull request by joseflauzino**
### Description

The unit tests of class `HostMetricsResponse` will fail when they are executed on machines with `locale` other than `en_US`. This happens because locales can have distinct decimal separators, for instance, `.` in `en_US` and `,` in `pt_BR`.

This PR solves the mentioned problem by modifying the tests to use the decimal separator character according to the Locale set on the local machine (which is running the tests), instead of using a hardcoded decimal separator.

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [x] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [x] Minor
- [ ] Trivial

### How Has This Been Tested?

I ran the unit tests of class `HostMetricsResponse` with locale `pt_BR` and `en_US`. In both cases, the tests passed.

### CODE REVIEW

**Review by DaanHoogland — APPROVED** [review]
code looks good

**Comment by DaanHoogland** [comment]
If packaging succeeds it compiles and the test is good. No testing needed as it is a unit test only.
@blueorangutan package

**Comment by DaanHoogland** [comment]
@joseflauzino if you want this in 4.16.1 please rebase. otherwise just wait for a second review and we are good to merge.

**Review by GutoVeronezi — APPROVED** [review]
Code LGTM

**Comment by joseflauzino** [comment]
@DaanHoogland we can keep in main (4.17).
Thanks

**Comment by GutoVeronezi** [comment]
Merging based on 2x LGTM (https://github.com/apache/cloudstack/pull/5713#pullrequestreview-815700415 and https://github.com/apache/cloudstack/pull/5713#pullrequestreview-815996701) and package results (https://github.com/apache/cloudstack/pull/5713#issuecomment-979016151).

**Pull request merged**
